>From bb0d8f93123feb6d9ea80e7edf0e612c8ac569ca Mon Sep 17 00:00:00 2001
From: James McCoy <jamessan@debian.org>
Date: Sat, 28 Jun 2014 23:26:23 -0400
Subject: [PATCH] Fix false positive when "compiling" python files
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Allow whitespace between the filename and the following “...” or end of
line.

Signed-off-by: James McCoy <jamessan@debian.org>
---
 bin/blhc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/blhc b/bin/blhc
index 2624585..72918fe 100755
--- a/bin/blhc
+++ b/bin/blhc
@@ -466,7 +466,7 @@ sub is_non_verbose_build {
     return 0 if $line =~ /^\s*C\+\+.+?:\s+(?:yes|no)\s*$/;
     return 0 if $line =~ /^\s*C\+\+ Library: stdc\+\+$/;
     # "Compiling" non binary files.
-    return 0 if $line =~ /^\s*Compiling \S+\.(?:py|el)['"]?(?:\.\.\.)?$/;
+    return 0 if $line =~ /^\s*Compiling \S+\.(?:py|el)['"]?\s*(?:\.\.\.)?$/;
     # "Compiling" with no file name.
     if ($line =~ /^\s*[Cc]ompiling\s+(.+?)(?:\.\.\.)?$/) {
         # $file_extension_regex may need spaces around the filename.
-- 
2.0.0

